docs/spec/interfaces.md

```diff
@@ -28,15 +28,15 @@ it previously indicated success.
 
 ## Callable
 
-A **Callable** resource represents an _Addressable_ endpoint which receives
-events and optionally returns events to forward downstream. One example of a
+A **Callable** resource represents an _Addressable_ endpoint which receives an
+event as input and optionally returns an event to forward downstream. One example of a
```

Comment:
The spec supports BATCH mode, we could support that in the future. _events_ is probably correct long term.

Reply:
I agree that batch mode might make sense, but this is very generic description of that a Callable receives _something_ and returns _something_ is not very helpful IMO. E.g. when a specification conformant Callable returns multiple events in batch-mode _now_, what would happen ? Does this mean that although a Callable is valid according to the spec (returning multiple events), it will still cause unwanted behaviour in the system ? Will this be fixed until GA (including trigger filtering for batched input events) ?

I think stating _we could support that in the future_ is too weak for a specification which is in effect **now** (or supposed to be for GA). Maybe we should think about a version scheme for the spec, too ? (and reserver the many case for a 2.0 spec ?)